math/mathcore/src/FitUtil.cxx

```diff
@@ -627,14 +633,17 @@ double FitUtil::EvaluateChi2Residual(const IModelFunction & func, const BinData
       }
       else {
          SimpleGradientCalculator  gc( npar, func);
-         if (!useBinIntegral )
+         if (!useBinIntegral ) {
             gc.ParameterGradient(x, p, fval, g);
-         else {
+            // std::cout << "compute numerical gradient for model func at " << x[0] << " ";
+            // for (unsigned int ip = 0; ip < npar; ip++) std::cout << "(" << p[ip] << ", " << g[ip] << ") " ;
+            // std::cout << std::endl;
```

Comment:
Can this commented out code be removed, or converted to some proper logging code?